# {"file_path": "xxx/stt.wav"}
string input_json
---
string output_json
# {"status": "success", "asr_result": txt}
# {"status": "error", "message": "xxx"}
